src/content/docs/apm/agents/java-agent/installation/java-install-overview.mdx

```diff
@@ -0,0 +1,175 @@
+    id="download-from-curl"
+    title={<>Download using <InlineCode>curl</InlineCode></>}
+  >
+    Complete the following:
+
+    1. Start a command-line session.
+    2. Change to a temporary directory where you can download the zip file.
+    3. Execute this `curl` command:
+
+       ```
+       curl -O https://download.newrelic.com/newrelic/java-agent/newrelic-agent/current/newrelic-java.zip
+       ```
+    4. Unzip `newrelic-java.zip`:
+
+    ```
+    unzip newrelic-java.zip
+    ```
+  </Collapser>
+
+  <Collapser
+    id="download-from-powershell"
+    title={<>Download using <InlineCode>Invoke-WebRequest</InlineCode> (PowerShell)</>}
+  >
+    Complete the following:
+
+    1. Start a PowerShell session.
+    2. Change to a temporary directory where you can download the zip file.
+    3. Execute this PowerShell command:
+
+       ```
+       Invoke-WebRequest -Uri https://download.newrelic.com/newrelic/java-agent/newrelic-agent/current/newrelic-java.zip -OutFile newrelic-java.zip
+       ```
+    4. Unzip `newrelic-java.zip`:
+
+       ```
+       Expand-Archive -Path newrelic-java.zip -DestinationPath <var>DESTINATION_PATH</var>
+       ```
+  </Collapser>
+
+  <Collapser
+    id="download-from-UI"
+    title="Download the agent"
+  >
+    From [Java agent release notes](/docs/release-notes/agent-release-notes/java-release-notes), download `newrelic-java.zip` to a temporary directory and unzip it.
+  </Collapser>
+</CollapserGroup>
+
+## 3. Configure the agent [#config-file]
+
+Here's how to do a basic setup with agent configuration settings in `newrelic.yml`.
+
+<Callout variant="tip">
+  For an advanced installation, you can pass settings with [environment variables](/docs/agents/java-agent/configuration/java-agent-configuration-config-file#Environment_Variables), [Java system properties](/docs/agents/java-agent/configuration/java-agent-configuration-config-file#System_Properties), or [server-side configuration](/docs/agents/manage-apm-agents/configuration/server-side-agent-configuration). To understand the precedence of these configuration settings, see [Java agent configuration: Config file](/docs/agents/java-agent/configuration/java-agent-configuration-config-file).
+</Callout>
+
+The Java agent requires the `license_key` and `app_name` settings at startup. All the other settings are optional, and you can review them in [Java agent configuration: Config file](/docs/agents/java-agent/configuration/java-agent-configuration-config-file).
+
+Go to the unzipped installation file, and inside the `newrelic` subdirectory, edit `newrelic.yml`:
+
+1. Find the line with `license_key`.
+2. Replace <var>'&lt;%= license_key %>'</var> with your [license](/docs/accounts/install-new-relic/account-setup/license-key). For example:
+
+   ```
+   license_key: 456n20n1367ov2s174v51nvn789d21s67v26NRAL
+   ```
+3. Find the line with `app_name`.
+4. Replace <var>My Application</var> with a name that helps you identify the application. For example:
```

Comment:
Should there be backticks around <var> stuff?

Reply:
I'm just restoring a pre-existing doc that I deleted a few weeks ago. Turns out there's some potentially important stuff in here.